### DISCUSSION TITLE
How can I get dom node?

### DISCUSSION THREAD

**Discussion by FatNerdPeng** [nuxt/nuxt#19184] [Questions] [answered]
Hello,I want to create a QRCode but failed like this:
![image](https://user-images.githubusercontent.com/33616855/220298839-372192c2-93d5-4ac3-b0e1-c3c9718c5389.png)
![image](https://user-images.githubusercontent.com/33616855/220299149-0eafc6eb-1a1c-46d5-8a8d-9f450d0efd1a.png)
![image](https://user-images.githubusercontent.com/33616855/220299345-f4865e62-8bdf-4fcf-9708-639c02d70611.png)
If I use "qrcodeRef.value" like this will receive error:
![image](https://user-images.githubusercontent.com/33616855/220299612-c57b7f22-29fe-418e-8ba5-982ef470a256.png)
![image](https://user-images.githubusercontent.com/33616855/220299743-04f2a8fa-2496-4af8-b840-3d1be1743f20.png)
I don't what did I wrong,thanks!

**Comment by TheAlexLichter** [comment, marked as answer]
Template refs only work after the component has been mounted. Do your call in an `onMounted` hook 👍

**Reply by FatNerdPeng** [reply]
thank you,it's ok
